bentoml/_internal/log.py

```diff
@@ -102,13 +108,25 @@ def format(self, record: logging.LogRecord) -> str:
             "tracebacks_show_locals": get_debug_mode(),
             "show_path": get_debug_mode(),  # show log line # in debug mode
         },
+        "build": {
```

Comment:
it's completely fine if the logging format look different for this, I'd prefer us not trying to maintain this code here

Reply:
noted